### ISSUE TITLE
training parameters

### ISSUE THREAD

**Issue by kizilkanat** [AlfredXiangWu/face_verification_experiment#26]
Hi, I want to train LightenedCNN_A from scratch, using a subset of CASIA-Webface, including 167K images. 
I want to apply learning parameters as close to as your setup.
What is the actual number of your training and validation images?
What is the training batch size, and validation batch size?
What is the total number of iterations?
What is the initial and final learning rate, and how does it change btw. (sigmoid weight decay, exponential weight decay, fixed, etc?) When and how does the learning rate decrease? (actual iteration number) 
Can you give all the parameters for LightenedCNN_A and LightenedCNN_B seperately?
Thanks in advance.

**Comment by AlfredXiangWu** [comment]
I randomly select 10,000 images from webface as the validation.
training batch size: 32
iteration: 2,000,000
The learning rate is 1e-3 at the beginning of training, and then I manually decrease the learning rate when the loss is not decreasing. The final learning rate is 1e-5.
The configuration of  B is similar to the configuration of model A.
